index.html

```diff
@@ -102,7 +102,7 @@ <h3><dfn>Stream</dfn> interface</h3>
 
       <pre class="idl">
       interface Stream : EventEmitter {
-        Quad read();
+        any read();
```

Comment:
Not sure if this is possible in IDL, but can a type parameter be added to `Stream` -> `Stream<T>`? This would make the semantics of _chunk_ a bit more meaningful.

Reply:
WebIDL defines it, but very specific for `Promise` or `sequence`, I didn't find anything if it can be used in general. If you would like to add it to the TypeScript definition, I think we can't avoid gaps in the WebIDL TypeScript mapping, but that should be OK as long as the description matches.

Reply:
Yes, I'll definitely add the type parameter to the TS definitions.

> but that should be OK as long as the description matches

I'm fine with that. Some text would be needed that explains the link between `chunk` and `Quad` in `Stream<Quad>`.

Reply:
@rubensworks I changed the text a little bit and added a description for the generic notation.